### DISCUSSION TITLE
Orange doesn't work when it is assigned to variable

### DISCUSSION THREAD

**Discussion by jpmadrigal07** [tailwindlabs/tailwindcss#9195] [Help] [answered]
Hi, I have this weird error that I encounter. When I'm trying to put the orange directly to the class, it works properly, but if I put on a variable, it doesn't work. I tried other color and put it in the variable and it is working.

![image](https://user-images.githubusercontent.com/20082606/187007103-78b79694-a083-434e-b7cc-38c7f1da8bde.png)
![image](https://user-images.githubusercontent.com/20082606/187007080-9b801d66-b769-4ed2-8d3f-65db7c63bcd0.png)
![image](https://user-images.githubusercontent.com/20082606/187007041-94379f3c-867d-4df1-8e53-61e460a1d1f3.png)

But orange works properly when I did this

![image](https://user-images.githubusercontent.com/20082606/187007245-06a50439-66d4-4af6-8783-5b15b69ac721.png)
![image](https://user-images.githubusercontent.com/20082606/187007285-4dcac9a9-2bdd-40af-958f-62976c0f25e5.png)
 
Thank you in advance.

**Comment by wongjn** [comment, marked as answer]
[As per the documentation](https://tailwindcss.com/docs/content-configuration#dynamic-class-names), one must write class names in full as-is. This is because Tailwind scans files for strings and does no language interpretation of its own, so in your case it wouldn't know about `bg-green-100` because all it sees is `bg-${color}-100` as a literal string.
